### ISSUE TITLE
Support for v2

### ISSUE THREAD

**Issue by ThouCheese** [NikosEfthias/paypal#2]
The current version used by this project is v2, but the most recent version is v4. Would you be interested in supporting that version as well? And is there any progress on this front?

**Comment by ta3pks** [comment]
I have been checking the PayPal docs but couldn't find anything relevant to the API versions. It even seemed to me that PayPal is dropping support for the API and moving to Braintree SDK. 
Can you suggest me the API doc source you have been checking? 
[here](https://developer.paypal.com/docs/integration/direct/payments/paypal-payments/#create-paypal-payment) I don't see much info about the version of the API.
If I can find proper documentation I can happily implement important parts of the v4

**Comment by ThouCheese** [comment]
You are completely right, I've got things mixed up, I meant supporting v2 instead of the current v1. Fixed it in the issue name!

**Comment by NikosEfthias** [comment]
and can you refer to v2 documentation, please? In Facebook API page there's a button to select API version for the docs but I could not find anything relevant for the PayPal API.

**Comment by ta3pks** [comment]
As a first step I decoupled the version from the urlString if and when i have time and energy hopefully i will complete the rest
